ai/STA/Tactic/demo_follow_robot.py

```diff
@@ -16,15 +16,20 @@
 class DemoFollowRobot(Tactic):
     def __init__(self, game_state: GameState, player: Player, p_target: Pose=Pose(), args: List[str]=None):
         super().__init__(game_state, player, p_target, args)
-        self.robot_to_follow_id = int(args[0])
+
+        # Since there no target provided, let's follow oneself
+        if args is None:
+            self.robot_to_follow_id = player.id
+        else:
+            self.robot_to_follow_id = int(args[0])
         self.current_state = self.halt
         self.next_state = self.halt
 
     def move_to_ball(self):
         self.status_flag = Flags.WIP
-        self.target = self.game_state.get_player(self.robot_to_follow_id).pose
+        self.target = self.game_state.get_player_position(self.robot_to_follow_id)
```

Comment:
pourquoi?, was fix in last PR

Reply:
Haha il y a 3 jours j'ai fait le même fix que toi, même si ce fix est incorrecte...